### PR TITLE
feat(console): api resource details

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

feat(console): api resource details

## Linear Issue Reference

* LOG-1695

## Testing

api-resource-details

### CODE REVIEW

File: packages/console/src/pages/ApiResourceDetails/index.tsx [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+  });
+
+  useEffect(() => {
+    if (!data) {
+      return;
+    }
+    reset(data);
+  }, [data, reset]);
+
+  const submit = useCallback(
```
Comment (wangsijie) [inline packages/console/src/pages/ApiResourceDetails/index.tsx:43]: unecessary useCallback?

File: packages/console/src/pages/ApiResourceDetails/index.tsx [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+                <FormField
+                  isRequired
+                  title="admin_console.api_resource_details.token_expiration_time_in_seconds"
+                  className={styles.textField}
+                >
+                  <TextInput {...register('accessTokenTtl', { required: true })} />
+                </FormField>
+              </div>
+              <div className={styles.submit}>
+                <Button
```
Comment (wangsijie) [inline packages/console/src/pages/ApiResourceDetails/index.tsx:103]: do we need a submitting state to prevent unexpected click?

**Review by wangsijie — CHANGES_REQUESTED** [review]

File: packages/console/src/pages/ApiResourceDetails/index.tsx (hunk 2 above)
Comment (wangsijie) [reply packages/console/src/pages/ApiResourceDetails/index.tsx:103]: can simply set to `disabled` while submitting

File: packages/console/src/pages/ApiResourceDetails/index.tsx (hunk 1 above)
Comment (xiaoyijun) [reply packages/console/src/pages/ApiResourceDetails/index.tsx]: updated

File: packages/console/src/pages/ApiResourceDetails/index.tsx (hunk 2 above)
Comment (xiaoyijun) [reply packages/console/src/pages/ApiResourceDetails/index.tsx:108]: Good idea.
updated.

**Review by wangsijie — APPROVED** [review]

**Pull request merged**